datasets/switchboard.py

```diff
@@ -215,6 +284,8 @@ def run(self):
         for rec_name, segs in sorted(rec_to_segs.items()):
             recording = corpus.Recording()
             recording.name = rec_name
+            if self.skip_empty_ldc_file and rec_name == "sw02167B":
```

Comment:
wouldn't it be better to just do this check before starting the loop and in case `skip_empty_ldc_file` is `True` just to eliminate it from the dictionary? Maybe not a relevant gain, but still better than doing the check multiple times within the loop.